components/cache/cache_pools.rst

```diff
@@ -282,7 +282,7 @@ item in the cache immediately (it returns ``true`` if the item was saved or
 ``false`` if some error occurred)::
 
     // ...
-    $userFriends = $cache->get('user_'.$userId.'_friends');
+    $userFriends = $cache->getItem('user_'.$userId.'_friends');
```

Comment:
It's just about saving cache items. 

I don't think it is necessary to get the value if you just want to save the value into cache.